### DISCUSSION TITLE
Redundancy  ld+json scripte on wire:navigate

### DISCUSSION THREAD

**Discussion by Misagh90** [livewire/livewire#7946] [1. Help]
Hi,
I use [artesaos/seotools](https://github.com/artesaos/seotools) for create my "ld+json scripte" but when i change pages in wire navigate  links, the ld+json script of the previous page is added to the head again
![Screenshot (954)](https://github.com/livewire/livewire/assets/6697222/14cd10fc-f728-4181-9aa7-22e3c5e9235a)
How should I solve this problem?

**Comment by danie-ramdhani** [comment]
are you using `@pushOnce` syntax? if so, it's not recommend using it in livewire spa version.
using `@assets` is the way to load your assets (javsacript or css) in livewire v3 (https://livewire.laravel.com/docs/javascript#loading-assets).

```blade
@pushOnce('head')
    <script src="/script.js"></script>
@endPushOnce
```

change to:

```blade
@assets
    <script src="/script.js"></script>
@endassets
```

**Reply by Misagh90** [reply]
no i'm just using these:
```
{!! SEOMeta::generate() !!}
{!! OpenGraph::generate() !!}
{!! Twitter::generate() !!}
{!! JsonLd::generate() !!}
```
and only "JsonLd" is repeated!
